docs/docs/firmware/can-traffic.md

```diff
@@ -1,38 +1,46 @@
----
-title: Calculating CAN traffic on a bus
----
 
+# Calculating CAN traffic
 
+In order to calculate the CAN load, we must declare a few variables and their importance within the CAN protocol.
 
-In order to calculate the CAN load, we must declare a few variables and their importance within the CANbus protocol. 
-
-1. Baud: (kbaud)
+## Baud (kbaud)
```

Comment:
Don't need the units here, an explanation of Baud is enough. Same with frequency